packages/editor/src/client/panel/authorEditPanel.tsx

```diff
@@ -220,7 +220,9 @@ export function AuthorEditPanel({id, onClose, onSave}: AuthorEditPanelProps) {
             </ListInput>
           </Panel>
           <Panel header={t('authors.panels.bioInformation')}>
```

Comment:
`<Panel>` has an option for an border. https://rsuitejs.com/components/panel/#%3CPanel%3E

Reply:
It won't work as it will add the frame around the whole panel including the title and not just the rich text block.